packages/admin/src/brevoConfiguration/BrevoConfigForm.tsx

```diff
@@ -60,25 +71,51 @@ export function BrevoConfigForm({ scope }: FormProps): React.ReactElement {
         variables: { scope },
     });
 
+    const {
+        data: doiTemplatesData,
+        error: doiTemplatesError,
+        loading: doiTemplatesLoading,
+    } = useQuery<GQLDoiTemplatesSelectQuery, GQLDoiTemplatesSelectQueryVariables>(doiTemplatesSelectQuery);
+
     const senderOptions =
         sendersData?.senders?.map((sender) => ({
             value: sender.email,
             label: `${sender.name} (${sender.email})`,
         })) ?? [];
 
-    const mode = data?.brevoConfig?.id ? "edit" : "add";
+    const doiTemplateOptions =
+        doiTemplatesData?.doiTemplates?.map((doiTemplate) => ({
+            value: doiTemplate.id,
+            label: `${doiTemplate.id}: ${doiTemplate.name}`,
+        })) ?? [];
 
     const initialValues = React.useMemo<Partial<FormValues>>(() => {
         const sender = sendersData?.senders?.find((s) => s.email === data?.brevoConfig?.senderMail && s.name === data?.brevoConfig?.senderName);
-        return sender
-            ? {
-                  sender: {
-                      value: sender.id,
+
+        const doiTemplate = doiTemplatesData?.doiTemplates?.find((template) => template.id === data?.brevoConfig?.doiTemplateId?.toString());
```

Comment:
I would avoid abbreviations and define it as doubleOptInTemplate

Reply:
0ae56569c0552e458995bba7816ab1b59b788816